File: mbrs/src/main/java/com/ftn/mbrs/service/KarticaService.java
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::Kartica


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.Kartica;

public interface KarticaService {

	public Kartica save(Kartica kartica);
	
	public Kartica update(Kartica kartica);

	public String delete(Long id);
	
	public Kartica findOne(Long id);
	
	public List<Kartica> findAll();
}

File: mbrs/src/main/resources/static/js/services/KarticaService.js
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::Kartica

angular.module('gremoNaElektrikoApp.KarticaService', [])
	.factory('KarticaService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/kartica');
		}
		
		factory.createKartica = function(kartica ) {
			return $http.post('/gremoNaElektriko/kartica', kartica);
		}

		factory.deleteKartica = function(id) {
			return $http.delete('/gremoNaElektriko/kartica/'+id);
		}
		
		factory.updateKartica = function(kartica) {
			return $http.put('/gremoNaElektriko/kartica', kartica);
		}
		
		return factory;
	});